### ISSUE TITLE
drag and drop list of possibilities

### ISSUE THREAD

**Issue by robbiedesutter** [gitana/alpaca#502]
Hi,

How can we create a fixed list of possibilities whereby we let the user order the items by drag and drop?

The use case is that we ask the user "please reorder the items in the list below by your preference" and we have a prepopulated list of items whereby he can drag an item to another position to create an orde of items that matters to him. 
A little bit like jqueryui.com/sortable/

thx for the hints

**Comment by jasondalycan** [comment]
Yes drag and drop sortable array fields would be a great feature addition.  Is this planned on the roadmap?
